modules/light-clients/07-tendermint/update_test.go

```diff
@@ -420,6 +420,43 @@ func (suite *TendermintTestSuite) TestUpdateState() {
 				suite.Require().False(found)
 			}, true,
 		},
+		{
+			"success with pruned consensus state using duplicate header", func() {
+				// this height will be expired and pruned
+				err := path.EndpointA.UpdateClient()
+				suite.Require().NoError(err)
+				pruneHeight = path.EndpointA.GetClientState().GetLatestHeight().(clienttypes.Height)
```

Comment:
maybe first grab the consensus state here before it's pruned?

Reply:
do you mean just to assert that it exists at the pruneHeight? 
I can push an update with that!

Reply:
done ✅